### PR TITLE
Minor changes to improve new/returning user experience

### PR DESCRIPTION
**Pull request by zhafen**
Changes:
* changed some parameter names
* changed some default values
* updated the README
* added a quickstart notebook
* updated `atlas_dir` setup

### CODE REVIEW

**Comment by nathimel** [comment]
Overall looks great, thanks for going through the codebase and reproducing a min working example! 

Just a few minor comments here and there, though I'm happy to merge and address them on my end.

I need to figure out what's up with the lint failing, I always get that error even though I run `black` on the root of the repo.

**Comment by zhafen** [comment]
Hope the lint part doesn't take too much time. I guess the linter is stricter than black is? (Speaking of, I hadn't heard of black. Looks nice.)

Regarding minor comments, happy to address them if you make the comments, but you're also welcome to take care of them. Your choice.

**Review by nathimel — APPROVED** [review]

**Comment by nathimel** [comment]
I merged! Thanks again for making the PR.

Yeah black is just my code formatter, I'm really not wild about it but it
can be nice for uniformity. I've heard yapf is pretty good too.

Btw, in your quickstart notebook, how quick (lol) was the iterate_expand? I
noticed you didn't set a num_pubs_max_per_exp, so it must have defaulted to
4000? How did that go?

File: src/examples/quickstart.ipynb [hunk 1]
```diff
@@ -86,7 +61,20 @@
    "execution_count": null,
    "metadata": {},
    "outputs": [],
-   "source": []
+   "source": [
+    "from sciterra.mapping.tracing import iterate_expand\n",
+    "\n",
+    "# Assuming the initial atlas contains just one publication\n",
+    "(atl.center, ) = atl.publications.keys()\n",
+    "# build out an atlas to contain 10,000 publications, with increasing dissimilarity to the initial publication, saving progress in binary files to the directory named \"atlas\".\n",
```
[the remark this answers is not in the document]
Comment (zhafen) [reply src/examples/quickstart.ipynb:69]: Ah, yeah. I'll change that.

File: src/sciterra/mapping/atlas.py
[the remark this answers is not in the document]
Comment (zhafen) [reply src/sciterra/mapping/atlas.py:1]: Sure, at least in regards to stating what we're doing. I recently set up pretty-advanced file-handling for one of my current projects, but when I moved it from local resources to AWS most of that work went out the window. I've subsequently become more cautious about QoL things like this.

**Comment by zhafen** [comment]
Thanks for merging!

Thanks for the tip about yapf, too.

Haha, the quickstart was decently fast (definitely less than an hour), but I didn't check exactly how long. As you pointed out, I didn't set a num_pubs_max_per_exp, but because I started from a single publication and I set the target_size to 1000 there were only ~3 expansions and none ever got close to 4000 new publications.

P.S. I'm now moving to play with the data and deep-learning. I'll make a new branch, but most of the experimentation will be in the scratch examples folder.

**Pull request merged**